h2o-core/src/main/java/hex/ModelExportOptions.java

```diff
@@ -0,0 +1,7 @@
+package hex;
+
+import water.api.StreamWriteOption;
+
+public enum ModelExportOptions implements StreamWriteOption {
```

Comment:
class name is usually singular

Reply:
thanks for catching the typo, it really looks bad "ModelExportOptions implements StreamWriteOption" :)